app/views/charges/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if current_user.premium? %>
+  <%= render partial: "charges/downgrade" %>
+<% else %>
+  <%= form_tag charges_path do %>
+    <h4>Upgrade To Premium</h4>
+    <script class='stripe-button' src="https://checkout.stripe.com/checkout.js"
+                                  data-key="<%= @stripe_btn_data[:key] %>"
+                                  data-amount=<%= @stripe_btn_data[:amount] %>
```

Comment:
in the action where you're trying to render this partial, have you defined `@stripe_btn_data`?